cfy_manager/components/base_component/base_component.py

```diff
@@ -43,17 +46,20 @@ def start(self):
         self.verify_started()
         self.logger.info('Component started')
 
-    def stop(self):
+    def stop(self, only_if_installed=False):
```

Comment:
Maybe call this 'force'- it'll make the if statement in a few line's time easier to read.